RNKeychainManager/RNKeychainManager.m

```diff
@@ -255,12 +255,13 @@ - (void)insertKeychainEntry:(NSDictionary *)attributes withOptions:(NSDictionary
   });
 }
 
-- (OSStatus)deletePasswordsForService:(NSString *)service
+- (OSStatus)deletePasswordsForService:(NSString *)service 
 {
   NSDictionary *query = @{
     (__bridge NSString *)kSecClass: (__bridge id)(kSecClassGenericPassword),
     (__bridge NSString *)kSecAttrService: service,
     (__bridge NSString *)kSecReturnAttributes: (__bridge id)kCFBooleanTrue,
+    (__bridge NSString *)kSecAttrSynchronizable: (__bridge id)kCFBooleanTrue
```

Comment:
Does this implicitly make everything synchronised?

Reply:
It seems like that but I might need a test to make sure it is